packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboard.tsx

```diff
@@ -105,12 +107,20 @@ export const CloudPulseDashboard = (props: DashboardProperties) => {
     Boolean(resourceList)
   );
 
+  if (isDashboardApiError) {
+    return renderErrorState('Failed to fetch the dashboard details');
+  }
+
+  if (isResourcesApiError) {
+    return renderErrorState('Failed to fetch Resources');
+  }
+
   if (isJweTokenError) {
-    return (
-      <Grid item xs>
-        <ErrorState errorText="Failed to get jwe token" />
-      </Grid>
-    );
+    return renderErrorState('Failed to get jwe token');
```

Comment:
Should "jwe" be capitalized? Also, since this is a user-facing message, maybe we should remove "jwe" altogether?
```suggestion
    return renderErrorState('Failed to get token.');
```